presidio-analyzer/README.MD

```diff
@@ -24,8 +24,14 @@ python -m spacy download en_core_web_sm
 ```
 
 ## Getting started
+### Running Presidio as an HTTP server
+```sh
+cd presidio-analyzer
+python app.py
+curl -d '{"text":"John Smith drivers license is AC432223", "language":"en"}' -H "Content-Type: application/json" -X POST http://localhost:3000/analyze
+```
```

Comment:
We might want to direct to use compose here later on. Even if it's just a single service, we can show how to run compose with one service only and analyzer as exampel

Reply:
i think running as a python package first is ok. but is there a difference between doing
python app.py and doing
flask run
?

Reply:
If it's not flask, the doc will be incorrect. 
By running Python app.py, you can ensure the app takes care of itself.

Reply:
From the exact reason, I would document the orchestrated way, app.py might change as well :)
Also takes care of env vars that might be needed later on, and since documented in presidio docs anyway, less confusing